### ISSUE TITLE
[win_chocolatey] Exception setting "changed": The property 'changed' cannot be found on this object. Verify that the property exist s and can be set.

### ISSUE THREAD

**Issue by guillaumededrie** [ansible/ansible#17139]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

module `win_chocolatey`
##### ANSIBLE VERSION

`ansible 2.1.1.0`
##### CONFIGURATION

Playbook launched using `Vagrant 1.8.5`.
##### OS / ENVIRONMENT

`Debian 8.5 amd64`
##### SUMMARY

In Ansible 2.1.1.0, using module `win_chocolatey` end up with an error, but using Ansible 2.0.1.0, I've got the expected result (see below).
##### EXPECTED RESULTS

```
  {"changed": true, "invocation": {"module_name": "win_chocolatey"}}
```
##### ACTUAL RESULTS

```
FAILED! => {"changed": false, "failed": true, "invocation": {"module_name": "win_chocolatey"}, "msg": "Exception setting \"changed\": \"The property 'changed' cannot be found on this object. Verify that the property exist
s and can be set.\""}
```

**Comment by nitzmahone** [comment]
Already fixed in devel and 2.1.2 via https://github.com/ansible/ansible-modules-extras/commit/0d8a7b875e7e59ef197297b7b6d79f83de4f7c29 (will be included in 2.1.2RC1 this/next week-ish)

**Comment by guillaumededrie** [comment]
thx @nitzmahone!
